app/views/errors/error_500.html.erb

```diff
@@ -0,0 +1,6 @@
+<% content_for :title do %>DARIA - 500<% end %>
+<% content_for :error_page do %>error<% end %>
+
+<h2>Sorry! Something went wrong. (500)</h2>
+
+<p>This is probably an us problem - try again later, and let us know if it persists.</p>
```

Comment:
Might be worth specifying who 'us' is here (Daria maintainers vs the Fund). Maybe:

```suggestion
<p>This is probably an us problem - try again later, and have your organization let the Daria maintainers know if it persists.</p>
```